docker/orca-nightly/Dockerfile

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG SPARK_VERSION=2.4.6
+ARG SPARK_HOME=/opt/spark
+ARG JDK_VERSION=8u192
+ARG JDK_URL=your_jdk_url
+ARG BIGDL_VERSION=2.3.0-SNAPSHOT
```

Comment:
what's the mechanism to modify this? manual modify?

Reply:
can pass args when build the docker.

e.g. `--build-arg SPARK_VERISION=3.1.3`